src/main/features/claimant-response/views/check-and-send.njk

```diff
@@ -80,7 +80,18 @@
     {%  if draft.formaliseRepaymentPlan.option.value === FormaliseRepaymentPlanOption.SIGN_SETTLEMENT_AGREEMENT.value %}
       {{ tableStart('settlementAgreement', 'Settlement agreement') }}
 
-      {% if draft.courtOfferedPaymentIntention.paymentOption === PaymentType.BY_SPECIFIED_DATE.value %}
+      {% if draft.courtOfferedPaymentIntention.paymentOption === PaymentType.IMMEDIATELY.value %}
+        {% set amountToPay = 'the full amount' %}
+        {% if claim.response.responseType == ResponseType.PART_ADMISSION.value %}
+          {% set amountToPay = claim.response.amount | numeral %}
+        {% endif %}
+        {{ row('The agreement', t('{{ defendantFullName }} will pay ' + amountToPay + ', immediately.
```

Comment:
Can we insert `amountToPay` by interpolation rather than concatenation?

Reply:
done